### PR TITLE
Authorizer throws exceptions accessing non-existent access tokens

### PR DESCRIPTION
**Pull request by tpavlek**
Fixes the problems Identified in #580

### CODE REVIEW

**Comment by vinkla** [comment]
Thanks for your contribution! This looks good to me. 

1. Please see my comments and make adjustment. 
2. Do we really need have the exception within `Exception` directory? IMO we should move this to the root.
3. Squash your commits.

**Comment by tpavlek** [comment]
Thanks for the feedback @vinkla - I took care of the changes and squashed.

**Comment by vinkla** [comment]
Thanks for updating it! Now it is up to @lucadegasperi to decide whether he wants to merge this or not.

File: src/NoActiveAccessTokenException.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
```
Comment (vinkla) [inline src/NoActiveAccessTokenException.php]: Don't forget to add the license.

Comment (tpavlek) [inline src/NoActiveAccessTokenException.php]: Thanks, It's added now. Everything should be up-to-snuff as soon as @lucadegasperi gives feedback :+1:

**Comment by lucadegasperi** [comment]
:+1:

**Comment by tpavlek** [comment]
@vinkla Addressed both pieces of your feedback.

With @lucadegasperi's :+1: now is this good to go? :)

**Comment by vinkla** [comment]
Thanks!

**Comment by dongkaipo** [comment]
I can still get this error in the new version.How can i fix it?
 When i use` dingo-api` to auth by `oauth2-server-laravel` i will get an error `Call to undefined method Closure::authenticate()` . 
And my colleagues will need this service, ORZ.....
Can anybody help me? Thanks! :)

**Comment by vinkla** [comment]
@dongkaipo Please open an issue for this.

**Comment by dongkaipo** [comment]
@vinkla thanks, it work now :)

**Comment by vinkla** [comment]
Okay, great! What was the issue?

**Comment by dongkaipo** [comment]
@vinkla thanks , but i get another error (https://github.com/dingo/api/issues/710) . I don`t know where the error configure

**Comment by dongkaipo** [comment]
@vinkla i delete all files and reinstall it , it work.

**Comment by vinkla** [comment]
Okay, thanks for sharing.

**Pull request merged**